lib/services/serviceBus/lib/servicebusservicebase.js

```diff
@@ -80,8 +80,8 @@ function ServiceBusServiceBase(configOrNamespaceOrConnectionStringOrSettings, ac
       }
     }
 
-    var endpoint = url.format({ protocol: 'https:', port: 443, hostname: configOrNamespaceOrConnectionStringOrSettings + '.' + ServiceClientConstants.CLOUD_SERVICEBUS_HOST });
-    var stsendpoint = url.format({ protocol: 'https:', port: 443, hostname: acsNamespace + '.' + ServiceClientConstants.CLOUD_ACCESS_CONTROL_HOST });
+    var endpoint = url.format({ protocol: 'https:', port: 443, hostname: configOrNamespaceOrConnectionStringOrSettings + '.' + this.host });
+    var stsendpoint = url.format({ protocol: 'https:', port: 443, hostname: acsNamespace + '.' + this.host });
```

Comment:
the above change is not required as the host is used in the next few lines in the right way